config/locales/en.yml

```diff
@@ -373,6 +374,9 @@ en:
       root: *root
     result:
       root: *root
+    admin:
+      internal_categories:
+        root: "Internal Categories"
```

Comment:
@muescha , I really forgot to add this key on others i18n files, I will provide this adjustment on next commit. Thanks! :smiley: